handler errors with serverless enterprise

User reported issue where the handler is being configured differently from how IOpipe sets it, and ends up with a module not found error.  

Error when invoking: 
```"errorType": "Error",
"errorMessage": "Cannot find module './/opt/nodejs/node_modules/@iopipe/iopipe.js'",
"trace": [
"Error: Cannot find module './/opt/nodejs/node_modules/@iopipe/iopipe.js'",
" at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)",
" at Module._require.o.require (/var/task/serverless_sdk/index.js:9:88803)",
" at require (internal/modules/cjs/helpers.js:22:18)",
" at Object.<anonymous> (/var/task/s_appsyncOdsResolver.js:12:23)",
" at Module._compile (internal/modules/cjs/loader.js:701:30)",
" at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)",
" at Module.load (internal/modules/cjs/loader.js:600:32)",
" at tryModuleLoad (internal/modules/cjs/loader.js:539:12)",
" at Function.Module._load (internal/modules/cjs/loader.js:531:3)",
" at Module.require (internal/modules/cjs/loader.js:637:17)"
]
}
```

Upon investigation, the handler was set incorrectly to `s_appsyncOdsResolver.handler` instead of `/opt/nodejs/node_modules/@iopipe/iopipe.handler`

Manually changing the handler via the AWS console to `/opt/nodejs/node_modules/@iopipe/iopipe.handler` fixed the issue.  

I suspect that AppSync plugin could be messing with the handler after we configure it.

--- a/README.md
+++ b/README.md
@@ -106,6 +106,10 @@ This plugin currently supports the following AWS runtimes:
 
 When using with [serverless-webpack](https://github.com/serverless-heaven/serverless-webpack) it is highly recommended to use the `nodejs10.x` runtime as this runtime does not require a helper script. For earlier Node.js runtimes a helper script is required which must be included via your `webpack.config.js`. The `serveless-iopipe-layers` plugin must follow the `serverless-webpack` plugin in your `serverless.yml`. See [here](https://github.com/iopipe/serverless-iopipe-layers/tree/master/examples/serverless-webpack) for an example.
 
+### Using with Serverless Enterprise
+
+When using with the [Serverless Enterprise Plugin](https://github.com/serverless/enterprise-plugin), because this plugin uses a similar method to wrap functions, a helper script must be used. If using with `serverless-webpack`, the helper script must be included in your `webpack.config.js` as described above.
+
 ## Limitations
 
 * Doesn't currently support local invocation

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "serverless-iopipe-layers",
-  "version": "0.3.4",
+  "version": "0.3.5",
   "description": "Serverless plugin to add IOpipe lambda layers.",
   "main": "dist/index.js",
   "files": [

--- a/src/index.ts
+++ b/src/index.ts
@@ -165,7 +165,11 @@ export default class IOpipeLayerPlugin {
   }
 
   private getHandlerWrapper(runtime: string, handler: string) {
-    if (["nodejs6.10", "nodejs8.10"].indexOf(runtime) !== -1) {
+    if (
+      ["nodejs6.10", "nodejs8.10"].indexOf(runtime) !== -1 ||
+      (runtime === "nodejs10.x" &&
+        _.get(this.serverless, "enterpriseEnabled", false))
+    ) {
       this.addNodeHelper();
       return "iopipe-wrapper.handler";
     }

